blackfootwords/scripts/initializedb.py

```diff
@@ -82,10 +82,11 @@ def main(args):
             sid, pages = Sources.parse(ref)
             refs[(vsid, sid)].append(pages)
         data.add(
-            common.Value,
+            models.Lemma,
```

Comment:
We now have to create instances of our derived table upon database initialization.